### ISSUE TITLE
VSCode Java Debugger throwing unknown error

### ISSUE THREAD

**Issue by AboDa7aM** [microsoft/vscode-java-test#1190]
When i try to run the test, it runs without any problems.
But When trying to debug a test this error is shown
```
'void
org.eclipse.jdi.internal.connect.ConnectorImpl$StringArgumentImpl.<init>(org.eclipse.jdi.i...
java.lang.String,java.lang.String, java.lang.String, boolean)'
```
![alt text](https://i.stack.imgur.com/yb4YQ.png "Error")

I've looked into the Output tab under "Language Support for Java" and its giving me this trace
```
[Error - 2:53:28 PM] Apr 30, 2021, 2:53:28 PM [error response][launch]: 'void org.eclipse.jdi.internal.connect.ConnectorImpl$StringArgumentImpl.<init>(org.eclipse.jdi.internal.connect.ConnectorImpl, java.lang.String, java.lang.String, java.lang.String, boolean)'
'void org.eclipse.jdi.internal.connect.ConnectorImpl$StringArgumentImpl.<init>(org.eclipse.jdi.internal.connect.ConnectorImpl, java.lang.String, java.lang.String, java.lang.String, boolean)'
java.lang.NoSuchMethodError: 'void org.eclipse.jdi.internal.connect.ConnectorImpl$StringArgumentImpl.<init>(org.eclipse.jdi.internal.connect.ConnectorImpl, java.lang.String, java.lang.String, java.lang.String, boolean)'
	at com.microsoft.java.debug.plugin.internal.AdvancedLaunchingConnector$AdvancedStringArgumentImpl.<init>(AdvancedLaunchingConnector.java:124)
	at com.microsoft.java.debug.plugin.internal.AdvancedLaunchingConnector.defaultArguments(AdvancedLaunchingConnector.java:47)
	at com.microsoft.java.debug.core.DebugUtility.launch(DebugUtility.java:205)
	at com.microsoft.java.debug.core.DebugUtility.launch(DebugUtility.java:97)
	at com.microsoft.java.debug.core.adapter.handler.LaunchWithDebuggingDelegate.launch(LaunchWithDebuggingDelegate.java:180)
	at com.microsoft.java.debug.core.adapter.handler.LaunchRequestHandler.launch(LaunchRequestHandler.java:242)
	at com.microsoft.java.debug.core.adapter.handler.LaunchRequestHandler.handleLaunchCommand(LaunchRequestHandler.java:142)
	at com.microsoft.java.debug.core.adapter.handler.LaunchRequestHandler.handle(LaunchRequestHandler.java:81)
	at com.microsoft.java.debug.core.adapter.DebugAdapter.lambda$dispatchRequest$0(DebugAdapter.java:89)
	at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1106)
	at java.base/java.util.concurrent.CompletableFuture.thenCompose(CompletableFuture.java:2235)
	at com.microsoft.java.debug.core.adapter.DebugAdapter.dispatchRequest(DebugAdapter.java:88)
	at com.microsoft.java.debug.core.adapter.ProtocolServer.dispatchRequest(ProtocolServer.java:118)
	at com.microsoft.java.debug.core.protocol.AbstractProtocolServer.lambda$new$0(AbstractProtocolServer.java:78)
	at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:60)
	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:61)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:52)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
The launch.json configuration was auto-generated, and previously worked. No changes have been made to my JDK or my pom.xml.

I also created a new test app and it had the same problem, so it's not limited to this app or workspace.

Any ideas of what is going on here? (I tried googling the... error... but got not anything helpful)

**Comment by AboDa7aM** [comment]
Someone else posted the same problem on StackOverFlow
[https://stackoverflow.com/questions/67327031/vscode-java-debugger-throwing-unknown-error](url)

**Comment by testforstephen** [comment]
this seems duplicated with #1189.

The "Debugger for Java" extension published a hotfix release 0.33.1 for it, did you try it?
